setup.py

```diff
@@ -2,7 +2,7 @@
 
 
 install_requires = [
-    "wagtail>=2.5,<2.6",
+    "wagtail~=2.0",
```

Comment:
```suggestion
    "wagtail>=2.5,<2.6",
```

Let's revert this change, especially since Wagtail will soon no longer be required in this repo.

Reply:
Suggested change is done. Another PR will remove requirements on wagtail in this repo.